src/hb-ot-math-table.hh

```diff
@@ -563,13 +563,11 @@ struct MathGlyphConstruction
     if (variants_count)
     {
       int64_t mult = font->dir_mult (direction);
-      hb_array_t<const MathGlyphVariantRecord> arr = mathGlyphVariantRecord.sub_array (start_offset, variants_count);
-      unsigned int count = arr.length;
-      for (unsigned int i = 0; i < count; i++)
-      {
-	variants[i].glyph = arr[i].variantGlyph;
-	variants[i].advance = font->em_mult (arr[i].advanceMeasurement, mult);
-      }
+      + mathGlyphVariantRecord.sub_array (start_offset, variants_count)
+      | hb_map ([&] (const MathGlyphVariantRecord &record) -> hb_ot_math_glyph_variant_t
+		{ return {record.variantGlyph, font->em_mult (record.advanceMeasurement, mult)}; })
+      | hb_sink (hb_array (variants, *variants_count))
```

Comment:
This works, but maybe make it like the other as well.

Reply:
Liked the previous one but also like to make it more consistent also, so, done